records/generator_perf_test.go

```diff
@@ -0,0 +1,37 @@
+package records
+
+import (
+	"math/rand"
+	"strconv"
+	"testing"
+)
+
+func BenchmarkInsertRR(b *testing.B) {
+	const (
+		clusterSize = 1000
```

Comment:
So, I don't think it matters to the actual count of these items, but I think most people run smaller clusters with more apps.